code/zFulpstationCode/fulp_overwrite_vars.dm

```diff
@@ -339,4 +339,54 @@
 //** FULPSTATION EMAG NERFS by Surrealistik Feb 2020 ENDS
 //---------------------------------------------------------------------------
 //** Makes the emag much more expensive and introduces a 6 TC budget emag
+//***************************************************************************
+
+
+//***************************************************************************
+//** FULPSTATION SYNDIBORG DISGUISE MENU by Surrealistik March 2020 BEGINS
+//---------------------------------------------------------------------------
+//** Lets medical and engineering syndiborgs choose a disguise from a list.
+//** Also tweaks the medical syndiborg, removing emag and granting more med items
+//***************************************************************************
+/obj/item/borg_chameleon //List of available disguises
+	var/disguise_text //for feedback
+
+/obj/item/robot_module/syndicate_medical
+	basic_modules = list(
+		/obj/item/assembly/flash/cyborg,
+		/obj/item/reagent_containers/borghypo/syndicate,
+		/obj/item/reagent_containers/borghypo,
+		/obj/item/shockpaddles/syndicate/cyborg,
+		/obj/item/healthanalyzer,
+		/obj/item/borg/apparatus/beaker,
+		/obj/item/reagent_containers/dropper,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/surgical_drapes,
+		/obj/item/retractor,
+		/obj/item/hemostat,
+		/obj/item/cautery,
+		/obj/item/surgicaldrill,
+		/obj/item/scalpel,
+		/obj/item/organ_storage,
+		/obj/item/melee/transforming/energy/sword/cyborg/saw,
+		/obj/item/holobed_projector/robot,
+		/obj/item/crowbar/cyborg,
+		/obj/item/extinguisher/mini,
+		/obj/item/pinpointer/syndicate_cyborg,
+		/obj/item/stack/medical/gauze/cyborg,
+		/obj/item/gun/medbeam,
+		/obj/item/borg/cyborghug/medical,
+		/obj/item/borg/lollipop,
+		/obj/item/borg_chameleon
+		)
+
+/obj/item/reagent_containers/borghypo/syndicate
+	desc = "An experimental piece of Syndicate technology used to produce powerful restorative nanites used to very quickly restore injuries of all types. Also synthesizes potassium iodide for radiation poisoning and lipolicide and anacea for stealthy kills and heparin for more overt ones."
```

Comment:
You do realize that potassium iodide DECREASES (not increases) your radiation level, right? It's a MEDICINE, not a poison.

Reply:
You do realize that's a default medicine for the syndihypo, right?

Reply:
Oh, I misinterpretted "synthesizes potassium iodide for radiation poisoning" as you saying that it was FOR CAUSING radiation poisoning, not you saying that it's FOR CURING radiation poisoning.

Maybe you could make that a bit more clear in the description?